plotly_resampler/aggregation/gap_handlers.py

```diff
@@ -78,7 +78,8 @@ def _get_gap_mask(self, x_agg: np.ndarray) -> Optional[np.ndarray]:
         med_diff, x_diff = self._calc_med_diff(x_agg)
 
         # TODO: this 4 was revealed to me in a dream, but it seems to work well
-        gap_mask = x_diff > 4 * med_diff
+        # After some consideration, we altered this to a 4.1
+        gap_mask = x_diff > 4.1 * med_diff
         if not any(gap_mask):
```

Comment:
I arrived at almost the same number when I implemented this..funny how that works

Reply:
Still seems to fail here and there (but not consistently), but already significantly fewer test fails than with `4` :upside_down_face: